### PR TITLE
Post gml

### PR DESCRIPTION
**Pull request by jread-usgs**
#277 #275 #274

### CODE REVIEW

**Comment by jread-usgs** [comment]
@lindsaycarr and @dblodgett-usgs can you review?

Dave, can you review the POST for GML.id filtering that is in values-webgeom.R? 

Lindsay, can you test this against your county workflow? This takes away the arbitrary 10,000 feature limit, and does the filtering on the server (instead of client side). I am worries that this approach although probably better, may be a lot slower than the `GET` all feature attribute values w/ client filtering that was done before.

File: R/values-webgeom.R [hunk 1]
```diff
@@ -45,20 +45,39 @@ setMethod(f = "values",signature="webgeom",
           }
 )
 
+wfsFilterFeatureXML <- function(.Object, knife=webprocess(), match.case = TRUE){
+  match.case.char <- ifelse(match.case, 'true','false')
+  top <- newXMLNode(name='wfs:GetFeature',
+                    attrs=c('service'="WFS",'version'= version(.Object),
+                            'xsi:schemaLocation' = paste(c(.Object@WFS_NAMESPACE,knife@WPS_SCHEMA_LOCATION),collapse=" ")),
+                    namespaceDefinitions=c('ogc' = knife@OGC_NAMESPACE,
```
Comment (dblodgett-usgs) [inline R/values-webgeom.R]: :100:! Super proper xml stuff makes me happy.

**Comment by dblodgett-usgs** [comment]
Looks great. See comment over in #277 regarding how you could use this.

**Comment by lindsaycarr** [comment]
Actually might be faster (at least for my query)

With your branch: 
```
   user  system elapsed 
  13.84    0.34   57.51 
```

With master:
```
   user  system elapsed 
  15.07    0.33   60.59 
```

**Comment by jread-usgs** [comment]
Looks like no changes needed to this. Merging

**Pull request merged**